mod/groups/lib/groups.php

```diff
@@ -613,3 +588,37 @@ function groups_prepare_form_vars($group = null) {
 
 	return $values;
 }
+
+/**
+ * Checks if the user is subscribed to group notifications
+ *
+ * @param \ElggGroup $group Group entity (defaults to page owner)
+ * @param \ElggUser  $user  User entity (defaults to logged in user)
+ * @return bool
+ */
+function groups_is_user_subscribed($group = null, $user = null) {
+
+	if (is_null($group)) {
+		$group = elgg_get_page_owner_entity();
+	}
+
+	if (is_null($user)) {
+		$user = elgg_get_logged_in_user_entity();
+	}
+
+	if (!$group instanceof \ElggGroup || !$user instanceof \ElggUser) {
+		return false;
+	}
+
+	$subscribed = false;
+	$NOTIFICATION_HANDLERS = _elgg_services()->notifications->getMethodsAsDeprecatedGlobal();
```

Comment:
I'm not sure there is an alternative, the notifications plugin uses this entirely.  The notifications upgrade from 1.8 => 1.9 was a big breaking change with this as a BC hack.